src/EditorControls.js

```diff
@@ -116,7 +116,9 @@ class EditorControls extends Component {
         if (this.props.beforeAddTrace) {
           this.props.beforeAddTrace(payload);
         }
-        graphDiv.data.push({type: 'scatter', mode: 'markers'});
```

Comment:
what's the reasoning for this change?

Reply:
a fit is a new trace in the data array, so I need to be able to have the new trace be something other than 'scatter'

Reply:
if you're actually not using this, let's revert it :)